modules/data-entry/src/main/frontend/src/dataHomepage/ExportButton.jsx

```diff
@@ -443,22 +448,39 @@ function ExportButton(props) {
                 </Grid>
               )}
               <FormControl variant="standard" fullWidth className={classes.variableDropdown}>
-                <InputLabel id="label">Select questions/sections from this questionnaire</InputLabel>
-                <Select
-                  variant="standard"
-                  labelId="label"
-                  value={tempValue}
-                  label="Select questions/sections from this questionnaire"
-                  onChange={handleEntitySelected}
-                >
-                  { entities?.filter(v => !selectedEntityIds.includes(v.path))
-                      .map(v =>
-                        <MenuItem value={v.path} key={`option-${v.name}`} className={classes.variableOption}>
-                          { getAvatar(v.type) }
-                          <ListItemText primary={v.name} secondary={v.text} />
-                        </MenuItem>)
+                <Autocomplete
+                  open={autocompleteOpen}
+                  onOpen={() => {
+                    setAutocompleteOpen(true);
+                  }}
+                  onClose={() => {
+                    setAutocompleteOpen(false);
+                  }}
+                  value={tempValue && variables.find(item => item.path == tempValue) || null}
+                  PopperComponent={FilterPopper}
+                  className={classes.answerField}
+                  getOptionLabel={(option) => option.text || option.name}
+                  options={entities?.filter(v => !selectedEntityIds.includes(v.path)) || []}
+                  renderOption={(props, option) =>
+                    <MenuItem
+                      value={option.path}
+                      key={option.path}
+                      className={classes.variableOption}
+                      onClick={(event) => {handleEntitySelected(option.path); setAutocompleteOpen(false);}}
```

Comment:
`props` are not used in this rendering, which means that any handling of mouse and keyboard events has been discarded. This prevents keyboard navigation as well as internal setting of the value. Instead of recording the selected entities on click on each option, allow the built-in event handling to take place and record the value change via Autocomplete's `onChange(event, value)` handler. 
```suggestion
                      {...props}
```

Reply:
Done this for ReferenceInput, see b460502 for inspiration.